### DISCUSSION TITLE
一些bug以及样式上的建议

### DISCUSSION THREAD

**Discussion by aozaki-kuro** [spencerwooo/onedrive-vercel-index#17] [General]
感谢作者一直坚持不懈开发新的onedrve-index项目！
目前我也尝试性地部署了项目，发现bug和样式想要修改的地方仍然较多。

测试页面：
https://suisei.vercel.app/%E2%9C%A83D%20Live

## 发现的Bug
1. `readme.md`加载时无法识别`html`代码，以及`fontawesome`似乎无法加载的样子。期望：`readme.md`内的`html`代码正确加载。
2. `onedrive-cf-index`时遗留的问题仍然存在：文件夹带`#`便无法获取正确路径。期望：带`#`的文件路径可以被自动替换为`%23`
3. 尽管我已经指定使用`monospace`字体，在标题超长时会出现这样未对齐的情况。期望：图标以及标题对齐。
![image](https://user-images.githubusercontent.com/28428951/124858589-f457df80-dfe0-11eb-95c4-fc9aeb0fb178.png)

## 想要修改的样式
1. 由于我所需要分享的内容标题较长，所以希望扩展整个边框尺寸，对`.bg-white.shadow.rounded`添加`<style="min-width: 70rem">`后发现整个框架向右扩展，而非`onedrive-cf-index`时居中向两侧扩展。如图所示：
![image](https://user-images.githubusercontent.com/28428951/124854554-1732c580-dfda-11eb-8058-898ab481bf9f.png)
由于我没有相关开发经验，希望作者能够指点一下正确扩展的姿势。
2. 由于我的项目不需要`Last Modified`，希望可以加入开关关闭这一栏。

谢谢！

**Comment by spencerwooo** [comment]
> 由于我的项目不需要 Last Modified，希望可以加入开关关闭这一栏。

要不你跟原项目 [#132](https://github.com/spencerwooo/onedrive-cf-index/issues/132) 打一架？（笑

**Reply by aozaki-kuro** [reply]
哈哈，打架还是免了，比较想知道怎么才能自己删。。。
